3p/integration.js

```diff
@@ -374,6 +375,7 @@ register('blade', blade);
 register('bodymovinanimation', bodymovinanimation);
 register('brainy', brainy);
 register('bringhub', bringhub);
+register('broadbandy', broadbandy);
```

Comment:
Please revert the changes in this file. It is not needed.

Reply:
hi we removed the requested line
Thanks